plugins/web-worker-offloading/hooks.php

```diff
@@ -63,36 +65,40 @@ function wwo_init(): void {
 		'before'
 	);
 
-	wp_add_inline_script( 'web-worker-offloading', $partytown_js );
+	$scripts->add_inline_script( 'web-worker-offloading', $partytown_js );
 }
-add_action( 'wp_enqueue_scripts', 'wwo_init' );
+add_action( 'wp_default_scripts', 'wwo_register_default_scripts' );
 
 /**
- * Marks scripts with `web-worker-offloading` dependency as async.
+ * Prepends web-worker-offloading to the list of scripts to print if one of the queued scripts is offloaded to a worker.
  *
- * This is needed because scripts offloaded to a worker thread can be considered async. However, they may include `before` and `after` inline
- * scripts that need sequential execution. Once marked as async, `filter_eligible_strategies()` determines if the
- * script is eligible for async execution. If so, it will be offloaded to the worker thread.
+ * This also marks their strategy as `async`. This is needed because scripts offloaded to a worker thread can be
+ * considered async. However, they may include `before` and `after` inline scripts that need sequential execution. Once
+ * marked as async, `filter_eligible_strategies()` determines if the script is eligible for async execution. If so, it
+ * will be offloaded to the worker thread.
  *
  * @since 0.1.0
  *
- * @param string[]|mixed $script_handles Array of script handles.
- * @return string[] Array of script handles.
+ * @param string[]|mixed $to_do An array of enqueued script dependency handles.
```

Comment:
why is `$script_handles` changed to `$to_do`?

Reply:
Ah, because this is the name of the param passed to `print_scripts_array`. But I can change it back.

Reply:
Reverted in bfb9e116a6c33cffbe729d5593e07448502ec1f2